snooty/test_parser.py

```diff
@@ -847,3 +847,31 @@ def test_cardgroup() -> None:
         </directive>
         </root>""",
     )
+
+
+def test_toctree() -> None:
```

Comment:
What is the plan for the toc,yaml files in the docs repo? If we have not addressed those in this test we might put a todo here. Otherwise LGTM.

Reply:
According to @i80and, we will run a one-time migration to convert those files into rST format. I believe we'll no longer support yaml, but in any case it's a separate issue from this ticket!